commit 5396a58cd6c2a4073febcc0d286e261be1e1d868
Author: Valentin David <me@valentindavid.com>
Date:   Sun Jun 22 10:30:56 2025 +0200

    linux: Add all PINCTRL config for Intel
    
    Some hardware, link touchpads, does not initialize properly without
    them.

diff --git a/files/linux/fdsdk-config.sh b/files/linux/fdsdk-config.sh
index da45395a7..8f47f4a58 100644
--- a/files/linux/fdsdk-config.sh
+++ b/files/linux/fdsdk-config.sh
@@ -1324,6 +1324,24 @@ case "$arch" in
         enable X86_AMD_PLATFORM_DEVICE
         enable CONFIG_X86_USER_SHADOW_STACK
         enable PINCTRL_INTEL_PLATFORM
+        module PINCTRL_ALDERLAKE
+        enable PINCTRL_BAYTRAIL
+        module PINCTRL_BROXTON
+        module PINCTRL_CANNONLAKE
+        module PINCTRL_CEDARFORK
+        module PINCTRL_CHERRYVIEW
+        module PINCTRL_DENVERTON
+        module PINCTRL_ELKHARTLAKE
+        module PINCTRL_EMMITSBURG
+        module PINCTRL_GEMINILAKE
+        module PINCTRL_ICELAKE
+        module PINCTRL_JASPERLAKE
+        module PINCTRL_LAKEFIELD
+        module PINCTRL_LEWISBURG
+        module PINCTRL_METEORLAKE
+        module PINCTRL_METEORPOINT
+        module PINCTRL_SUNRISEPOINT
+        module PINCTRL_TIGERLAKE
     ;;
 esac
 
